src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -47,18 +49,28 @@ std::unique_ptr<CascadiaSettings> CascadiaSettings::LoadAll(const bool saveOnLoa
     {
         const auto actualData = fileData.value();
 
-        // If Parse fails, it'll throw a hresult_error
-        JsonObject root = JsonObject::Parse(actualData);
+        // Parse the json data.
+        Json::Value root;
+        std::unique_ptr<Json::CharReader> reader{ Json::CharReaderBuilder::CharReaderBuilder().newCharReader() };
+        const auto raw = winrt::to_string(actualData);

```

Comment:
aaaaaaaaaaaaaaaaaaaaaaaaa

we're loading the file from utf-8
converting to utf-16
back to utf-8

Reply:
good catch